components/package.json

```diff
@@ -27,7 +27,7 @@
     "peerDependencies": {
         "react": "^16.13.0",
         "react-native": "^0.63.0",
-        "react-native-paper": "^4.0.0",
+        "react-native-paper": "^4.7.1",
```

Comment:
I see what you meant now. I would leave this at 4.0.0, but just update the yarn-lock file to have the latest version (via yarn upgrade). Unless we are expressly using features that were not available until 4.7, users should be fine to use anything over 4.